src/components/pages/auth/utils.ts

```diff
@@ -1,3 +1,9 @@
+import {
+  UiNode,
+  UiNodeInputAttributes,
+  UiNodeInputAttributesTypeEnum,
+} from '@ory/client'
```

Comment:
putting these imports in this file is increasing bundlesize by 18kb (unfortunately, tree shaking is not able to shake code across different import trees)

Solution would be to move changeButtonTypeOfSSOProvider into a separate file and making it a separate module.